src/modules/prompt-finder/hole-patching-prompt.json

```diff
@@ -0,0 +1,5 @@
+{
+    "system_prompt": "You are an AI assistant that makes educated guesses.  ",
+    "user_prompt": "You are provided with a prompt with integrated variables. these variables are contained within ellipses {}. For a specific variable x, generate a placeholder value, and respond with ONLY  a Json containing that value. YOUR ANSWER MUST BE A VALID JSON. \n\n  For example, for the prompt \"Hi my name is {{name}}\" and the variable is name, you answer would be \"{variable: name, value: Jack} \".\n\n The prompt you need to process is {{prompt_with_holes}}, and the variable is {{variable}}.\n to better inform your guess, you should use the following source code as context: \n `{{source_code_file}}` \n",

```

Comment:
@dhia-e-rzig one suggestion is including the quotation marks in the exemplar json value. 

Also anytime I do json in prompts, I double check how it is tokenizing because the combination of "} and {" can be lead to very infrequent tokens. Not sure if we have this issue here, but it could be worth putting through the tiktoken tokenizer.

Reply:
Thanks for the suggestions! I will update the prompt accordingly. 
I am using this online tool to check (https://tiktokenizer.vercel.app/) and I think it looks okay ( it's encoding {{ as one token, which has a much higher "rank" than { ), although I am no expert on this 😅 so please let me know what you think! I can also add a few tests that automatically run the tokenizing on the prompts within the project to alert us if there are any low frequency tokens, although I'm not sure which cutoff we want to use.